src/solver/infeasible-problem-analysis/report.h

```diff
@@ -7,6 +7,8 @@
 #include "constraint.h"
 #include "ortools/linear_solver/linear_solver.h"
 
+using namespace operations_research;
+
```

Comment:
Adding this at the top of header file is considered bad practice

Reply:
In this case, I replaced **operations_research::MPVariable** with **MPVariable**.
As the second is clear enough, and the first cumbersome, I  find this better.
I will change back if you insist.

Reply:
ok, I will.